styles/globals.css

```
@@ -29,3 +29,33 @@ textarea {
 .MuiDialogContent-root {
   padding: 0 20px;
 }
+
+.loading-bar {
+  height: 5px;
```

Seems like 3px is enough.

```suggestion
  height: 3px;
```

---

Copy that